app/src/hooks/initialize-agent.ts

```diff
@@ -190,7 +190,7 @@ const useInitializeBCAgent = () => {
     
     if (store.preferences.usePushNotifications) {
       logger.info('Activating push notifications...')
-      await activate(newAgent)
+      activate(newAgent)
```

Comment:
I'm hoping this reverted line will fix the issue Shel mentioned that happens during SauceLabs app initialization